health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/components/AddColumnsWrapper.js

```
@@ -97,24 +106,32 @@ const AddColumnsWrapper = ({ formData, onSelect, props: customProps }) => {
                     onClose={() => {
                         setShowDeletePopup(false);
                     }}
                 ></PopUp>
             )}
+            {showToast && (
+                <Toast
+                    label={showToast?.message}
+                    type={showToast?.type}
+                    isDleteBtn={true}
+                    onClose={() => setShowToast(null)}
+                ></Toast>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)